deploy/crds/metalkube_v1alpha1_baremetalhost_cr.yaml

```diff
@@ -4,4 +4,7 @@ metadata:
   name: example-baremetalhost
 spec:
   # Add fields here
-  size: 3
+  bmc:
```

Comment:
probably need a "type" field somewhere in the future.  "ipmi" vs "idrac" etc

Reply:
Btw, as this evolves, can you update the docs repo too?  There's a section at the bottom about the API.

https://github.com/metalkube/metalkube-docs